src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -536,4 +536,12 @@ public uint GetPaginationLimit(int? first)
             return defaultPageSize;
         }
     }
+
+    /// <summary>
+    /// Retrieve the value of LogLevel from runtime
+    /// </summary>
+    public Level? LogLevelValue()
+    {
+        return Runtime?.LogLevel?.Value;
```

Comment:
In the startup class I see 2 the following two helpers which both return the type `LogLevel`:
- `public static LogLevel GetLogLevelBasedOnMode(...)`
- `public static LogLevel GetLogLevelBasedOnLevel(...)`

What do you think about moving log level determination logic to `RuntimeConfig`?
e.g. `LogLevel GetLogLevel()` which, internally, will check whether dab is in development mode -> returns correct log level and handles casting of `Level` to `LogLevel`.

On that same note, this function returns `Level`. Are there components of DAB which use `Level` directly or should the `RuntimeConfig` object's public API only return `LogLevel` types?

Reply:
The only components that uses `Level` is `LogLevelOptions` so `RuntimeConfig` objects should only return `LogLevel` types.

Reply:
There was more to this comment than identifying which components use `Level` directly. 

Any thoughts on this:
In the startup class I see 2 the following two helpers which both return the type LogLevel:

> - `public static LogLevel GetLogLevelBasedOnMode(...)`
> - `public static LogLevel GetLogLevelBasedOnLevel(...)`
> What do you think about moving log level determination logic to `RuntimeConfig`?
> e.g. `LogLevel GetLogLevel()` which, internally, will check whether dab is in development mode -> returns correct log level and handles casting of `Level` to `LogLevel`.